src/main/java/org/apache/commons/collections4/list/AbstractLinkedList.java

```diff
@@ -529,6 +532,7 @@ protected void addNode(final Node<E> nodeToInsert, final Node<E> insertBeforeNod
      * @throws NullPointerException if <code>node</code> is null
      */
     protected void removeNode(final Node<E> node) {
+        Objects.requireNonNull(node,"The new node to remove must not be null.");
```

Comment:
Please add a space after the comma.

Reply:
Thanks for your suggestions, I will fix the commit right away.

Reply:
Thanks for your suggestions, I will fix the commit right away.